templates/page.html

```diff
@@ -45,6 +45,35 @@
     {{- macros::meta_post(page=page, config=config) }}
     {%- endif %}{%- endif %}
 
+
+    {%- if page.extra.section %}
+    
+      {%- set tag = get_taxonomy_term(kind="tags", term=page.extra.section, lang=lang) %}
+      {%- set page_num = tag.page_count | as_str %}
+
+        {%- if lang != config.default_language %}
+          {%- set parts_string = trans(key="Series_parts", lang=lang) | replace(from="$NUMBER_OF_PARTS", to=page_num) %}
+        {% else %}
+          {%- set parts_string = page_num ~ " part Series" %}
+        {% endif -%}
+
+
+
+    <div class="series-mobile">
+      <h4><a href="{{ tag.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}"></a>{{ tag.name }} ({{ parts_string }})</a></h4>
+      <ol>
+      {%- for inner in tag.pages | sort(attribute="year") %}
+      {%- if inner.content == page.content %}
+      <li><a href="{{ inner.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}"><b>{{ inner.title }}</b></a></li>
+      {% else %}
+      <li><a href="{{ inner.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}">{{ inner.title }}</a></li>
+      {%- endif %}
+      {%- endfor %}
+      </ol>
```

Comment:
Is there a way to make this into a block so the code doesn't have to repeated below?

Reply:
I wouldn't worry about that for now, its ok for some duplicated code, especially in the templates since once the site is generated it wont affect performance.